### PR TITLE
test(agw): Adding new TCs for testing EPS bearer context IE in TAU

### PR DESCRIPTION
**Pull request by pruthvihebbani**
Signed-off-by: Pruthvi Hebbani <pruthvi.hebbani@radisys.com>

test(agw): Adding new TCs for testing EPS bearer context IE in TAU. https://github.com/magma/magma/issues/9194

## Summary

Adding new TCs for testing EPS bearer context IE in TAU

## Test Plan

Verified the new TCs are passing on multiple runs without introducing any memory leaks.

Note: This PR has dependency on s1ap tester PR: https://github.com/facebookexperimental/S1APTester/pull/64

### CODE REVIEW

File: lte/gateway/python/integ_tests/s1aptests/test_eps_bearer_context_status_ded_bearer_deact.py [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+
+    def tearDown(self):
+        """Cleanup"""
+        self._s1ap_wrapper.cleanup()
+
+    def test_eps_bearer_context_status_multi_ded_bearer_deact(self):
+        """Attach a single UE. Create 2 secondary PDNs and add 2
+        dedicated bearers to each of the secondary PDNs.
+        Send EPS bearer context status
+        IE in TAU request with all default bearer i.e bearer ids
```
Comment (ulaskozat) [inline lte/gateway/python/integ_tests/s1aptests/test_eps_bearer_context_status_ded_bearer_deact.py]: i.e -> i.e.,

Comment (ulaskozat) [reply lte/gateway/python/integ_tests/s1aptests/test_eps_bearer_context_status_ded_bearer_deact.py]: can we break the symmetry here and delete 1 dedicated bearer from one PDN and 2 from the other? Also, why don't we have dedicated bearer for the primary PDN?

File: lte/gateway/python/integ_tests/s1aptests/test_eps_bearer_context_status_ded_bearer_deact.py [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+            "request for UE id ",
+            ue_id,
+        )
+        tau_req = s1ap_types.ueTauReq_t()
+        tau_req.ue_Id = ue_id
+        tau_req.type = s1ap_types.Eps_Updt_Type.TFW_TA_UPDATING.value
+        tau_req.Actv_flag = True
+        # Set default bearers 5,6 and 9 as active and
+        # dedicated bearers 7,8,10 and 11 as inactive
+        # epsBearerCtxSts IE is 16 bits
```
Comment (ssanadhya) [inline lte/gateway/python/integ_tests/s1aptests/test_eps_bearer_context_status_ded_bearer_deact.py:217]: For readability, could you please add a reference to the specification and the bitmap for the EPS Bearer Context Status value used in line 218?

File: lte/gateway/python/integ_tests/s1aptests/test_eps_bearer_context_status_def_bearer_deact.py [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+        """Initialize"""
+        self._s1ap_wrapper = s1ap_wrapper.TestWrapper()
+        self._spgw_util = SpgwUtil()
+
+    def tearDown(self):
+        """Cleanup"""
+        self._s1ap_wrapper.cleanup()
+
+    def test_eps_bearer_context_status_def_bearer_deact(self):
+        """Attach a single UE. Create 2 secondary PDNs and add 2
```
Comment (ssanadhya) [inline lte/gateway/python/integ_tests/s1aptests/test_eps_bearer_context_status_def_bearer_deact.py]: Fix the comment, only 1 dedicated bearer is getting added for each secondary PDN in line 97-142. 
There are total of 5 bearers as seen in line 153.

File: lte/gateway/python/integ_tests/s1aptests/test_eps_bearer_context_status_def_bearer_deact.py [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+            "request for UE id ",
+            ue_id,
+        )
+        tau_req = s1ap_types.ueTauReq_t()
+        tau_req.ue_Id = ue_id
+        tau_req.type = s1ap_types.Eps_Updt_Type.TFW_TA_UPDATING.value
+        tau_req.Actv_flag = True
+        # Set bearer 5 as active in UE
+        # epsBearerCtxSts IE is 16 bits
+        tau_req.epsBearerCtxSts = 0x20
```
Comment (ssanadhya) [inline lte/gateway/python/integ_tests/s1aptests/test_eps_bearer_context_status_def_bearer_deact.py:188]: Please add reference to spec and bitmap for this IE

**Review by ssanadhya — APPROVED** [review]
LGTM, with minor comments

**Comment by ulaskozat** [comment]
@pruthvihebbani looks like there is a minor comment to be addressed. @ssanadhya are we pending on that to merge this PR?

**Comment by ssanadhya** [comment]
> @pruthvihebbani looks like there is a minor comment to be addressed. @ssanadhya are we pending on that to merge this PR?

Yes, once the comments are updated, we can merge

File: lte/gateway/python/integ_tests/s1aptests/test_eps_bearer_context_status_ded_bearer_deact.py (hunk 1 above)
Comment (pruthvihebbani) [reply lte/gateway/python/integ_tests/s1aptests/test_eps_bearer_context_status_ded_bearer_deact.py]: I will add a new TC for this scenario

**Pull request merged**